pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -129,7 +129,15 @@ func (jm *CronJobController) syncAll() {
 	glog.V(4).Infof("Found %d groups", len(jobsBySj))
 
 	for _, sj := range sjs {
-		syncOne(&sj, jobsBySj[sj.UID], time.Now(), jm.jobControl, jm.sjControl, jm.podControl, jm.recorder)
+		t, err := getCurrentTimeInZone(&sj)
```

Comment:
Use more descriptive variable names, like tzTime. It makes reading code so much easier and more obvious :)